cargo-dist/src/backend/ci/github.rs

```diff
@@ -661,13 +661,32 @@ fn github_runner_for_target(
     let result = Some(match target_triple.operating_system {
         OperatingSystem::Linux => runner_to_config(GithubRunnerRef::from_str("ubuntu-20.04")),
         OperatingSystem::Darwin => runner_to_config(GithubRunnerRef::from_str("macos-13")),
-        OperatingSystem::Windows => runner_to_config(GithubRunnerRef::from_str("windows-2019")),
+        OperatingSystem::Windows => {
+            // Default to cargo-xwin for Windows cross-compiles
+            if target_triple.architecture != Architecture::X86_64 {
+                cargo_xwin()
+            } else {
+                runner_to_config(GithubRunnerRef::from_str("windows-2019"))
+            }
+        }
         _ => return Ok(None),
     });
 
     Ok(result)
 }
 
+fn cargo_xwin() -> GithubRunnerConfig {
+    GithubRunnerConfig {
+        runner: GithubRunnerRef::from_str("ubuntu-20.04").to_owned(),
+        host: targets::TARGET_X64_LINUX_GNU.to_owned(),
+        container: Some(cargo_dist_schema::ContainerConfig {
+            image: ContainerImage::from_str("messense/cargo-xwin").unwrap(),
```

Comment:
we might consider at least an expect here on the offchance this ever fails (unlikely but still)

Reply:
Good call. Turns out there's a safe, unwrap/expect-free options here, so I went for that.